massgov/mayflower
# DP-27916 a11y home search

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)


## Description


### Main Nav
Modify roles for proper setup for buttons on the top menus.

### Search
Make the search components consistent with their attributes.
Add `autocomplete="off"` and `inputmode="search"`. 

## Related Issue / Ticket

- [JIRA issue](https://massgov.atlassian.net/browse/DP-27916)
- [Github issue]()

## Steps to Test


1. Go to home page.
2. Check the main nav markup for the top menu:
      - `li` with its child `button` has `role="menuitem"`
      - `li`'s child `button` has `role="button"`
      - `li` with its child `a` has `role="none"`
      - `li`'s child `a` has `role="menuitem"`
4. Check the search components in the header and the banner.  Find they have the red underlined attributes. 

## Screenshots
<img width="1444" alt="Mass_Gov" src="https://user-images.githubusercontent.com/9633303/233165682-fd5e67ad-1999-4d0a-a8c5-80381bedfcdb.png">


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


*

#### @TODO


*

#### Today I learned...
